lib/Event/Model/WebsiteSettingEvent.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Event\Model;
+
+use Pimcore\Model\WebsiteSetting;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class WebsiteSettingEvent extends Event
```

Comment:
We could make the class final and set the property to private instead of protected.

Reply:
Sounds like a good idea!